docs/resources/logdna_alert.md

```diff
@@ -83,11 +83,12 @@ resource "logdna_alert" "my_alert" {
 
 ## Import
 
-Preset Alerts can be imported by `id`, which can be found in the URL when editing the
-Preset Alert in the web UI:
+Preset Alerts can be imported by `presetid`, which can be found using the [List Preset Alerts API](https://docs.logdna.com/reference/list-alerts):
+- `Method`: GET
+- `URL`: `<API_URL>`/v1/config/presetalert?servicekey=`<SERVICE_KEY>`
```

Comment:
I believe, ever since migrating to OpenAPI, our public-facing documentation explicitly omits API key authentication via query params because it's the least secure option. We should change the auth scheme here to show supported HTTP headers (1) or Basic Auth (2). 

1. Custom HTTP Headers - `servicekey: <SERVICE_KEY>` || `apikey: <SERVICE_KEY>`
2. Basic Auth - `Authorization: Basic <encodeInBase64(credentials)>`
    - `credentials` is a string composed of + formatted as `<username>:<password>`, our usage here entails substituting `SERVICE_KEY` as the username and leaving the password blank
    - The colon separator should still included in the resulting string `<SERVICE_KEY>:`

Reply:
It makes sense. I thought to give customers an easiest way to call API and provide a link to our documentation where they can find more advanced ways to call our API.